dev/tasks/conda-recipes/r-arrow/build.sh

```diff
@@ -5,13 +5,14 @@ export DISABLE_AUTOBREW=1
 
 # arrow uses C++17
 export ARROW_R_CXXFLAGS="${ARROW_R_CXXFLAGS} -std=c++17"
+export LIBARROW_BUILD=false
 
 if [[ "${target_platform}" == osx-* ]]; then
     # See https://conda-forge.org/docs/maintainer/knowledge_base.html#newer-c-features-with-old-sdk
     export ARROW_R_CXXFLAGS="${ARROW_R_CXXFLAGS} -D_LIBCPP_DISABLE_AVAILABILITY"
 fi
 
-export PKG_CONFIG_PATH="${PREFIX}/lib/pkgconfig${PKG_CONFIG_PATH:+:${PKG_CONFIG_PATH}}"
+export PKG_CONFIG_PATH="${BUILD_PREFIX}/lib/pkgconfig${PKG_CONFIG_PATH:+:${PKG_CONFIG_PATH}}"
```

Comment:
This doesn't look right. `BUILD_PREFIX` is the build system, not host. It has been failing in the cross-compiling case and this change makes me suspicious whether we then link to the wrong libraries.

Reply:
Ah, sorry. This is not related to fix the R problem.
We may be able remove this line entirely. I'll try it.